### ISSUE TITLE
Error at robotDevastation init (XInitThreads)

### ISSUE THREAD

**Issue by David-Estevez** [asrob-uc3m/robotDevastation#1]
### Description

Sometimes, when robotDevastation is started, the program crashes, returning an error related to XInitThreads and SDL.

This error is hard to reproduce, because it does not happen always, but sporadically.
### Related links

Some info (that could be) related to this issue:

http://lazyfoo.net/SDL_tutorials/lesson33/

**Comment by David-Estevez** [comment]
Seems to be related to threads being used: 

> I was able to make it work by calling XInitThreads before anything else was called [2]

More info:
 * [1] http://stackoverflow.com/questions/31952711/threading-pyqt-crashes-with-unknown-request-in-queue-while-dequeuing
 * [2] http://en.sfml-dev.org/forums/index.php?topic=14853.0

**Comment by David-Estevez** [comment]
A fix was submitted at 81d19f8, consisting in calling XInitThreads in the SDLInputManager constructor. 

The code is now to be tested to check if the error disappeared or not.

**Comment by jgvictores** [comment]
Still occurs, Ubuntu 14.04 machine.

**Comment by David-Estevez** [comment]
Crash occurs at either:

* Line 34, RateThreadOutput.cpp:     
```cpp
display = SDL_SetVideoMode(640, 480, 32, SDL_HWSURFACE | SDL_DOUBLEBUF);  // SDL_SWSURFACE
```
* Or line 64, RdSDLInputManager.cpp:
```cpp
while(SDL_PollEvent(&event ) )
```

**Comment by David-Estevez** [comment]
I am going to assume issue #71 solved this at  27b941dd116ba1e

If this happens again, we will reopen the issue
